### ISSUE TITLE
twisted.internet.tcp.Connector.getDestination has invalid epytext in it

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5572]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#5572|
|Type|enhancement|
|Created|2012-03-27 13:02:51Z|
|Branch|https://github.com/twisted/twisted/tree/quick-doc-fix-5572|

This made it into the [#5085](https://github.com/twisted/twisted/issues/5085) merge due to some deficiencies in the way that new pydoctor errors are identified.

<details><summary>Searchable metadata</summary>

```
trac-id__5572 5572
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_quick_doc_fix_5572 branches/quick-doc-fix-5572
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1332853371000000 1332853371000000
changetime__1332854715000000 1332854715000000
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

(In [34021]) Branching to 'quick-doc-fix-5572'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph removed owner|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/quick-doc-fix-5572) should be available... eventually.  Should be easy points.

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_3"></a>|@ralphm set owner to @glyph|
|-|-|

Great! Builds pass. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

(In [34023]) Merge quick-doc-fix-5572.

Fix a minor documentation regression introduced by the previous commit.

Author: glyph

Reviewer: ralphm

Fixes: [#5572](https://github.com/twisted/twisted/issues/5572)
